docs/maps/import-geospatial-data.asciidoc

```diff
@@ -6,6 +6,30 @@ To import geospatical data into the Elastic Stack, the data must be indexed as {
 Geospatial data comes in many formats.
 Choose an import tool based on the format of your geospatial data.
 
+[discrete]
+[[import-geospatial-privileges]]
+=== Security privileges
+
+The {stack-security-features} provide roles and privileges that control which users can upload files.
+You can manage your roles, privileges, and
+spaces in the **{stack-manage-app}** in {kib}. For more information, see
+{ref}/security-privileges.html[Security privileges],
+<<kibana-privileges, {kib} privileges>>, and <<xpack-kibana-role-management, {kib} role management>>.
+
+To upload GeoJSON files in {kib} with *Maps*, you must have:
+
+* `all` {kib} privilege for `Maps`
```

Comment:
Maps & Index Pattern Management should be in bold, not courier

Also, should add an ending period after the first two bullet items to match the last two.